server/src/http/feature_refresher.rs

```diff
@@ -277,6 +277,7 @@ impl FeatureRefresher {
     pub async fn start_streaming_features_background_task(
         &self,
         app_name: String,
+        instance_id: String,
         custom_headers: Vec<(String, String)>,
```

Comment:
Ooh, this is a little sticky again. Having two very similar arguments with very similar values of the same type next to each other is a little dodgy. It'll be easy to flip them around. Can we chuck 'em in a config struct or use the type system to differentiate them?